### DISCUSSION TITLE
How would I set the server render distance with Minestom?

### DISCUSSION THREAD

**Discussion by dhruviyengar** [Minestom/Minestom#1115] [Help] [answered]
The title is pretty self-explanatory.

**Comment by GreatWyrm** [comment, marked as answer]
You set them with system properties
`minestom.chunk-view-distance` for chunks, and `minestom.entity-view-distance`.

To set these system properties, you can either pass them in as command line arguments when you run your server, or you can do `System.setProperty(name, value)`.

**Reply by muelleel** [reply]
Is there somewhere a list of all properties minestom is using?

Didn't find anything in the Javadoc or the Wiki...

**Reply by KrystilizeNevaDies** [reply]
> Is there somewhere a list of all properties minestom is using?
> 
> Didn't find anything in the Javadoc or the Wiki...

There is not a list, but that is a good idea, could you make an issue on the wiki github for us? Thanks.

**Reply by muelleel** [reply]
https://github.com/Minestom/wiki/issues/37
